frontend/views/top_containers/bulk_operations/_bulk_action_templates.html.erb

```diff
@@ -4,4 +4,90 @@
     <li><strong>${container[1]}</strong>&nbsp;&nbsp;&nbsp;${container[0]}</li>
   {/for}
   </ul>
---></div>
\ No newline at end of file
+--></div>
+
+<div id="bulk_action_update_ils_fields"><!--
+    <div class="alert alert-info" id="alertBucket">Click the checkbox of the fields to update.</div>
+    <%= form_tag({:controller => :top_containers, :action => :bulk_operation_update}, {:class => "form-horizontal", :id => "bulk_update_form", :onchange => "formChanged();" }) do |f| %>
+      <div class="span4">
+        <div class="control-group">
+          <label class="control-label" for="ils_holding_id">ILS Holding ID</label>
+          <div class="controls">
+            <%= text_field_tag "ils_holding_id", params["ils_holding_id"], :style => "width: 80%;", :disabled => "disabled", :id => "ils_holding_id" %>
+            <%= check_box_tag "update_ils_holding_id", params["update_ils_holding_id"], false, :onclick => "handleClick(this);", :name => "fieldToggle" %>
+          </div>
+        </div>
+        <div class="control-group">
+          <label class="control-label" for="ils_item_id">ILS Item ID</label>
+          <div class="controls">
+            <%= text_field_tag "ils_item_id", params["ils_item_id"], :style => "width: 80%;", :disabled => "disabled", :id => "ils_item_id" %>
+            <%= check_box_tag "update_ils_item_id", params["update_ils_item_id"], false, :onclick => "handleClick(this)", :name => "fieldToggle" %>
+          </div>
+        </div>
+        <div class="control-group">
+          <div class="controls">
+            <%= submit_tag "Update ${selection.length} records", :id => "bulk_update_submit", :class => "btn btn-primary", :disabled => "disabled" %>
+          </div>
+        </div>
+      </div>
+    <% end %>
+
+    <script>
+      $('#bulk_update_form').on("submit", function(event) {
+        event.preventDefault();
+        var ids = '';
+        {for container in selection}
+          ids += "${container[0]}".replace(/(.+\/)/, '') + ',';
+        {/for}
```

Comment:
If we're going to post IDs instead of URIs, maybe we should just use the ID as the checkbox value?  We could store the URI as a data attribute on the checkbox just in case we ever need that.